### ISSUE TITLE
Nice project

### ISSUE THREAD

**Issue by WouterJN** [pommi/python-itho-wpu#1]
Nice project! Are you using this with a Raspberry Pi? How is the stability?

How did you connect this electrical? Dit you use a level shifter?

Thank you in advance!

**Comment by pommi** [comment]
Thanks Wouter! I have to say, I got a lot of help from the author of https://github.com/ootjersb/pislave. I'm indeed running this on a Raspberry Pi (2b). It's still very experimental. The wiring is done as explained on https://github.com/ootjersb/pislave

**Comment by WouterJN** [comment]
Are you sure that the wiring diagram is correct? When I measure the DC voltage between Pin 3 (green/white) and Pin 5 (blue/white) of the RJ45 connector on my ITHO WPU I measure 16V instead of the 5v that I'm expecting.

**Comment by pommi** [comment]
It should be correct now. Initially the wiring was wrong. It was fixed here: https://github.com/ootjersb/pislave/commit/97246f89
Also in this forum post someone confirms a higher voltage: https://www.circuitsonline.net/forum/view/message/2071843#2071843
I measured 18V. In my case it was important to connect pin 5 to the HV pin of the level shifter (this wasn't in the diagram before). Without it, it didn't work for me.

**Comment by WouterJN** [comment]
Strange, that the +15v of the WPU is connected to the High Voltage of the level shifter. Normally level shifters are used to connect 3.3V I2C with 5V I2C.
In the forum post on circuit online ,the user 'rustyx'  warned for the 15v on the RJ45 connector.  It is not fully clear to me which voltage he used as HV for his level shift.

**Comment by WouterJN** [comment]
Today I emailed the poster of the message on the circuitonline.net forum (Rustyx), he did not connect the HV of the level shifter as the Itho WPU already have two pull up resistors on the 5v side. Finally I decided to not connect the HV of the level shifter and everything works great.

I strongly advice to not connect the 15-18v connector to HV as it is really not good to use such hight voltage as pull-up. 

<img width="665" alt="Screenshot 2020-12-09 at 19 29 22" src="https://user-images.githubusercontent.com/26854058/101671931-7be26d80-3a55-11eb-96df-899b81203151.png">

**Comment by pommi** [comment]
Thanks for sharing this information! Around 2 months ago, when I finally got it to work (with the help of @ootjersb) I remember it didn't work for me without having pin 5 attached to HV. But I just disconnected it and it still works (for now). I assumed it didn't work because of a wrong reference voltage on the HV. I'm not sure what the voltage is on the SDA/SCL pins, I wasn't able to measure that.

Good to see your output! It looks like you have a slightly different WPU model, because the values after (and including) "Kamertemperatuur" seem to be slightly off. The "getdatalog" action is currently hardcoded to my WPU version. I'm currently working on supporting multiple WPU models actually. My last commit (e05f25077e5) should fix the "Buitentemperatuur" btw.
